airbyte-cdk/bulk/toolkits/load-low-code/build.gradle

```diff
@@ -0,0 +1,34 @@
+import org.jsonschema2pojo.SourceType
+
+plugins {
+    id 'com.github.eirnym.js2p' version "1.0"
+}
+
+jsonSchema2Pojo {
```

Comment:
Do we really need this? This has been overall a pretty big pain in the platform, I'd rather avoid it if possible.

Reply:
An extra layer of build, dependencies. Now that we are kotlin, we also lose the null safety because the objects generated are java.